src/Clock.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string
+};
+
+type State = {
+  date: Date
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    date: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ date: new Date() });
+      // eslint-disable-next-line no-console
+      console.info(this.state.date.toUTCString().slice(-12, -4));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    if (prevProps.name !== this.props.name) {
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions, no-console
+      console.debug(`Renamed from ${prevProps.name} to ${this.props.name}`);
+    }
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerId);
+  }
+
+  render() {
+    const { date } = this.state;
+    const { name } = this.props;
+
+    const datePrint = date.toUTCString().slice(-12, -4);
+
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
+          {name}
+        </strong>
+
+        {' time is '}
```

Comment:
You don't need any parentheses here
```suggestion
        </strong>
        	time is
```